bridges/prometheus/example_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus_test
+
+import (
+	"go.opentelemetry.io/contrib/bridges/prometheus"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func ExampleNewMetricProducer() {
+	// Create a Promethes bridge "Metric Producer" which adds metrics from the
+	// [prometheus.DefaultGatherer]. Add the [WithGatherer] option to add metrics
```

Comment:
```suggestion
	// prometheus.DefaultGatherer. Add the WithGatherer option to add metrics
```

These are just regular code comments. The "Go Doc" syntax does not work here.